### DISCUSSION TITLE
[Subscriptions] Prevent character names from being "unlocked" via the unlock name service

### DISCUSSION THREAD

**Discussion by UnePomme** [chromiecraft/chromiecraft#7020] [Suggestions]
Paid subscription levels should prevent your character names from going "inactive".
If an account has an active paid subscription, the account should be labeled as "active" for the duration.

https://www.chromiecraft.com/product/name-unlock/

**Comment by Helias** [comment]
I implemented it, thanks for the idea!

this feature is available in all subscriptions now https://www.chromiecraft.com/en/tiers-subscription/
